pyccl/tests/test_cosmology.py

```diff
@@ -110,12 +110,6 @@ def test_cosmology_init():
         m_nu_type='normal')
 
 
-def test_cosmology_setitem():
-    cosmo = ccl.CosmologyVanillaLCDM()
-    with pytest.raises(NotImplementedError):
-        cosmo['a'] = 3
-
-
```

Comment:
Why did we lose this?

Reply:
I removed `__setitem__` because Python's native message is good, and we don't want to test the default internal workings of Python.